src/main/java/codeu/model/data/User.java

```diff
@@ -33,6 +36,9 @@
     private final Instant creation;
     private String bio;
     private Boolean admin;
+    private ArrayList<String> pals;
```

Comment:
I would declare these as List; use interface where ever possible

Reply:
yes, I will fix that!